src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -201,8 +201,8 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     maxBandwidthKbps: number
   ): void {
     if (this.isAndroid && this.isPixel3) {
-      this.videoWidth = Math.ceil(width / 32) * 32;
-      this.videoHeight = Math.ceil(height / 32) * 32;
+      this.videoWidth = Math.ceil(width / 64) * 64;
+      this.videoHeight = Math.ceil(height / 64) * 64;
```

Comment:
Some nitpick: Could you make this encapsulate under DefaultBrowserBehavior following the "requires" pattern ? Android Pixel3 may not be the only device we need this alignment.